src/pages/index.js

```diff
@@ -0,0 +1,277 @@
+import { setButtonText, deleteButtonText } from "../utils/helpers.js";
+
+const editModalBtn = document.querySelector(".profile__edit-button");
+const cardModalBtn = document.querySelector(".profile__new-post-button");
+const profileName = document.querySelector(`.profile__name-main`);
+const profileDescription = document.querySelector(`.profile__name-description`);
+
+const profileAvatar = document.querySelector(`.profile__avatar`);
+const avatarModalBtn = document.querySelector(`.profile__avatar-btn`);
+
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarModalCloseBtn = avatarModal.querySelector(".modal__close-btn");
+const avatarFormElement = avatarModal.querySelector(`.modal__form`);
+const avatarModalLinkInput = avatarModal.querySelector(`#profile-avatar-input`);
+const avatarSaveBtn = avatarModal.querySelector(`.modal__button`);
+
+const deleteModal = document.querySelector("#delete-modal");
+const deleteForm = deleteModal.querySelector(".modal__form");
+const deleteModalCloseBtn = deleteModal.querySelector(".modal__close-btn");
+
+const allModals = [...document.querySelectorAll(".modal")];
+
+const editModal = document.querySelector("#edit-modal");
+const editModalCloseBtn = editModal.querySelector(".modal__close-btn");
+const editModalNameInput = editModal.querySelector(`#profile-name-input`);
+const editModalDescriptionInput = editModal.querySelector(
+  `#profile-description-input`
+);
+const editFormElement = editModal.querySelector(`.modal__form`);
+const editSaveBtn = editModal.querySelector(`.modal__button`);
+
+const cardModal = document.querySelector("#add-card-modal");
+const cardModalCloseBtn = cardModal.querySelector(".modal__close-btn");
+const cardFormElement = cardModal.querySelector(`.modal__form`);
+const cardSubmitBtn = cardModal.querySelector(`.modal__button`);
+const cardModalLinkInput = cardModal.querySelector(`#card-link-input`);
+const cardModalNameInput = cardModal.querySelector(`#card-link-input-caption`);
+
+const previewModal = document.querySelector("#preview-modal");
+const previewModalImage = previewModal.querySelector(".modal__image");
+const previewModalPicDescription = previewModal.querySelector(
+  ".modal__pic-description"
+);
+const previewModalCloseBtn = previewModal.querySelector(
+  ".modal__close-btn_type_preview"
+);
+
+const cardTemplate = document.querySelector(`#card-template`);
+const cardsList = document.querySelector(`.cards__list`);
+
+let selectedCard, selectedCardId;
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "3e36594c-9f58-43ca-8212-d4a4b67e7c65",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([cards, userInfo]) => {
+    cards.forEach(function (ArrayItem) {
+      const cardElement = getCardElement(ArrayItem);
+      cardsList.prepend(cardElement);
+    });
+    profileAvatar.src = userInfo.avatar;
+    profileAvatar.alt = userInfo.avatar;
```

Comment:
`profileAvatar.alt = userInfo.avatar;` is not needed because you set the right `alt`  in line 79